src/components/Primary-Button/Button.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react';
+import { describe, it, expect } from 'vitest';
+
+import { PrimaryButton } from './Button';
+
+describe('PrimaryButton', () => {
+  it('renders the button with text', () => {
+    render(<PrimaryButton>Click me</PrimaryButton>);
+    expect(screen.getByText('Click me')).toBeInTheDocument();
```

Comment:
Test uses getByText to assert the presence of the button. Prefer accessible queries such as getByRole('button', { name: /text/i }) which better reflect how users interact with the UI and follow testing best practices.

**Sugerencia:**
```typescript
Replace with:
expect(screen.getByRole('button', { name: /Click me/i })).toBeInTheDocument();
```